pkg/operator/ceph/config/store.go

```diff
@@ -76,6 +99,9 @@ func (s *Store) CreateOrUpdate(clusterInfo *cephconfig.ClusterInfo) error {
 		members[i] = m.Name
 		i++
 	}
+
+	/* TODO: both of below need to go to secret to support CSI? */
```

Comment:
If the mons need to go in a secret for csi, i'd recommend generating it separate from what we do for the daemons. It would be nice if we only had to worry about the minimal set for clients, such as the mon endpoints.